shell/packages/sandstorm-ui-powerbox/powerbox-client.js

```diff
@@ -161,6 +161,28 @@ SandstormPowerboxRequest = class SandstormPowerboxRequest {
           cardData.title = "Admin: grant all outgoing network access";
         } else if (cardData.frontendRef.ipInterface) {
           cardData.title = "Admin: grant all incoming network access";
+        } else if (cardData.frontendRef.emailVerifier) {
+          const services = cardData.frontendRef.emailVerifier.services;
+          if (services) {
+            const name = services[0];
```

Comment:
It's somewhat of a temporary UI restriction that we don't currently support selecting multiple services. E.g. I might want to trust Google and email but not Github, or something. Perhaps in the future this can be accomplished through whatever multi-select UI we come up with for the powerbox, or perhaps what I really should have done here is made the first selection be "verify based on login provider" which would then have a custom-UI second step where the user chooses providers. I decided it wasn't worth doing now but figured there was no harm in making the underlying representation support it, so that we can easily twiddle the UI later.